packages/jest-config/src/getMaxWorkers.ts

```diff
@@ -15,8 +15,8 @@ export default function getMaxWorkers(
     return 1;
   } else if (argv.maxWorkers) {
     // TODO: How to type this properly? Should probably use `coerce` from `yargs`
-    const maxWorkers = (argv.maxWorkers as unknown) as number | string;
-    const parsed = parseInt(maxWorkers as string, 10);
+    const maxWorkers = argv.maxWorkers;
+    const parsed = parseInt(<string>maxWorkers, 10);
```

Comment:
```suggestion
    const parsed = parseInt(maxWorkers as string, 10);
```

I prefer this syntax for type casts (probably because I write React all day where the syntax you went for is not used)

Reply:
10-4, will keep it consistent 👍